packages/teleterm/src/ui/DocumentGateway/DocumentGateway.tsx

```diff
@@ -135,14 +135,20 @@ export function DocumentGateway(props: State) {
         Connect with GUI
       </Text>
       <Text>
-        To connect with a GUI database client, see our{' '}
+        To connect with a GUI database client, configure the client to connect
```

Comment:
I'm not a fan o this repetition:
<img width="256" alt="image" src="https://user-images.githubusercontent.com/20583051/175969953-862d945f-1318-4aee-9c5a-c3ed4d5b02a6.png">

What would you say for removing the repeated part completely? So it would be just `Configure the client to connect to host localhost on port 64307.` 

I'd also change `See our documentation` to `See the documentation`. Idk, it seems more professional to me :)

Reply:
Yeah, I noticed that repetition too, I think initially the copy was taken from the Web UI. I'll change it to "Configure the GUI database client to connect…".